### PR TITLE
Kestrel

### PR DESCRIPTION
**Pull request by No-Dad-Not-The-Belt**
## About the PR

Once used as a local shipping lane freighter, the Kestrel has been retrofitted to accommodate a medium (3-4) sized salvage crew. Capable of debris salvage and mining operations with or without the assistance of a Prospector, This vessel has seen widespread adoption by most of the salvage sector.

***Big thanks to @MatthewBadger on Discord for the sketch/idea that turned into the Kestrel and @Kesiath for the design inputs.**

**Media**

Kestrel for PR

:cl:
- adds : The NT-Kestrel
- adds: NT-Kestrel NF Shipyard Prototype file.

### CODE REVIEW

File: Resources/Prototypes/_NF/Shipyard/kestrel.yml
Comment (No-Dad-Not-The-Belt) [inline Resources/Prototypes/_NF/Shipyard/kestrel.yml:1]: Obsolete - updated price in commit 0f21037 (Price was set to 10,000 for testing)

**Comment by Cheackraze** [comment]
i like the layout generally of the ship, but I feel having both the cheap price and expedition capabilities will make it the only shuttle purchased for that purpose, and I wanted to keep the expedition shuttles up >50k price tag minimum.  Instead I think remove expeditions and you could lean heavier onto the prospectors-big-brother theme and keep it at around 40k but give it some more mining/salvaging amenities, like maybe half the parts to a ripley or something.  The extended access loading bay in the back could make this the premier salvage/mining vessel on frontier for the mid-size category.  I'll post my design suggestions below, but fundamentally I don't want to have an exped-capable ship that's under 50k right now

**Comment by Cheackraze** [comment]
image
something like this could make it an amazing salvaging ship imo

**Comment by Cheackraze** [comment]
image
here we go
something like this I could see being an excellent salvage vehicle

**Comment by Cheackraze** [comment]
Also please only 1 food/drink vending machine, you can make the other either into a shady cigs/job wardrobe vending machine or a booz-o-mat to go along with the mini-bar

**Comment by No-Dad-Not-The-Belt** [comment]
So if I'm reading the picture right replace the AME with a different engine, change atmospherics, add the additional cargo space and remove expeditions/add livery? 

Ill get started on the updates now, should I submit a new PR for the updated version or just update this branch?

**Comment by Cheackraze** [comment]
yea it can use uranium gens instead, and that frees up room to put your mini atmos there and put a little rp flavor in the corner and a window to see out of.  And also the floor tiling, you could change it up so it doesn't match the airlock entrance, otherwise yea you pretty much read my critique, I see this being a great salvage ship

**Comment by No-Dad-Not-The-Belt** [comment]
> yea it can use uranium gens instead, and that frees up room to put your mini atmos there and put a little rp flavor in the corner and a window to see out of. And also the floor tiling, you could change it up so it doesn't match the airlock entrance, otherwise yea you pretty much read my critique, I see this being a great salvage ship

Went ahead and made the changes and sent a commit for the new ship .yml. The picture above it accurate. Thanks for the help, and the constructive criticism!

**Review by Cheackraze — APPROVED** [review]

**Pull request merged**